src/en/contributing/websocket_http_api.md

```diff
@@ -68,11 +68,13 @@ These go wherever there is a `sort` field. The available sort types are:
 
 ## Default Rate limits
 
-These can be edited in your `lemmy.hjson` file.
+These can be edited in your [lemmy.hjson](https://github.com/LemmyNet/lemmy/blob/main/config/defaults.hjson) file.
```

Comment:
"These can be edited in your `lemmy.hjson` file, by copying the relevant section from [defaults.hjson](https://github.com/LemmyNet/lemmy/blob/main/config/defaults.hjson)."